database/migrations/2020_03_03_050629_create_folders_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateFoldersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('folders', function (Blueprint $table) {
+            $table-> increments('id');
+            $table-> string('title',20);
```

Comment:
メソッド呼び出す時、プロパティにアクセスするときにはスペースを入れないようにしましょう
```suggestion
            $table->increments('id');
            $table->string('title',20);
```